src/main/webapp/index.html

```diff
@@ -10,8 +10,25 @@ <h1 class="name">Events</h1>
     <div class="list">
     </div>
 
-    <div class="add">
+    <div id="addButton">
```

Comment:
I think it might be better if you used hyphens (Ie. add-button).  I think that's the mainstream for naming in HTML/CSS!

Reply:
SGTM!